src/app/features/visitor-form/visitor-form.component.scss

```diff
@@ -0,0 +1,32 @@
+@use '@angular/material' as mat;
+@import '../../styles/palettes';
+
+.mat-mdc-raised-button {
+    border-radius: 20px;
+}
+
+.form{
+    display: flex;
+    flex-direction: column;
+    padding: 8px;
+    max-width: 200px;
+    gap: 8px;
+    .form__input{
+        padding: 4px;
+        box-shadow: 2px 1px 1px rgba(0, 0, 0, 0.12);
+        border: none;
+    }
+    .form__button{
+        background:mat.get-color-from-palette($blue-palette, 700);
+        margin:16px;
```

Comment:
Tutaj powinna być spacja po dwukropku, masz włączonego prettiera? Powinien to poprawić

Reply:
Dodawajmy też spację między selektorem a nawiasem klamrowym